cloud-sql/postgres/activerecord/Gemfile

```diff
@@ -2,16 +2,17 @@ source "https://rubygems.org"
 git_source(:github) { |repo| "https://github.com/#{repo}.git" }
 
 # Bundle edge Rails instead: gem 'rails', github: 'rails/rails'
-gem "rails", "~> 5.2.3"
+gem "rails", ">= 5"
 # Use postgres as the database for Active Record
-gem "pg", "~> 0.21.0"
+gem 'pg', '~> 1.3', '>= 1.3.5'
 # Use Puma as the app server
 gem "puma", "~> 3.11"
 # Use SCSS for stylesheets
 gem "sass-rails", "~> 5.0"
 # Use Uglifier as compressor for JavaScript assets
 gem "uglifier", ">= 1.3.0"
 # See https://github.com/rails/execjs#readme for more supported runtimes
+# Use MiniRacer for deploying to App Engine Flexible
```

Comment:
MiniRacer should not be necessary, as nodejs is installed as part of the base image.

Reply:
Removed comment and removed related instruction from README